### PR TITLE
Adding Mui Checkbox Component

### PR DESCRIPTION
**Pull request by Momen-a9s**
relates to: https://anynines.atlassian.net/browse/INT-1234

Screenshot 2021-07-07 at 13 13 37

### CODE REVIEW

File: src/components/inputs/Checkbox/Checkbox.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+import { CheckboxProps } from '@material-ui/core';
+import { Checkbox } from './Checkbox';
+import WbSunnyIcon from '@material-ui/icons/WbSunny';
+import Brightness3Icon from '@material-ui/icons/Brightness3';
+
+export default {
+  title: '💧 Atoms/Checkbox',
+  component: Checkbox,
+  argTypes: {
+		id: {
```
Comment (r3p0x) [inline src/components/inputs/Checkbox/Checkbox.stories.tsx:13]: indentation looks really bad here

File: src/components/inputs/Checkbox/Checkbox.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  decorators: [withDesign],
+  parameters: {
+    controls: { expanded: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/FquPS1rVsEsTOPxR8SCw04/%F0%9F%8E%A8-Design-System?node-id=181%3A375',
+    },
+  },
+};
+// TODO: - adding examples with FormControlLabel
```
Comment (r3p0x) [inline src/components/inputs/Checkbox/Checkbox.stories.tsx:89]: love it

File: src/components/inputs/Checkbox/Checkbox.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+export const Default = Template.bind({});
+Default.args = {
+  id: 'checkbox-default'
+};
+
+export const Primary = Template.bind({});
+Primary.args = {
+  id: 'checkbox-Primary',
+	checked: true,
```
Comment (r3p0x) [inline src/components/inputs/Checkbox/Checkbox.stories.tsx:104]: Also indentation here, guess when sabbir is done with prettier and eslint we wont have to talk about that anymore ^^ Until then double check :P

File: src/components/inputs/Checkbox/Checkbox.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import {
+		Checkbox as MuiCheckbox,
```
Comment (r3p0x) [inline src/components/inputs/Checkbox/Checkbox.tsx:2]: omg here you got like 8 spaces indentation, don't you have that configured locally?

**Review by r3p0x — APPROVED** [review]
looks good except the indentation all over the place :thum

**Comment by r3p0x** [comment]
@Momen-a9s  Looks good overall just get your indentation setup straight

**Comment by Momen-a9s** [comment]
@r3p0x indentation is fixed now

**Pull request merged**